src/lib/y2partitioner/dialogs/main.rb

```diff
@@ -40,7 +43,13 @@ def run(system, initial)
         res = nil
         loop do
           res = super()
-          break if res != :redraw
+
+          next if res == :redraw
+          if res == :abort && Yast::Mode.installation
```

Comment:
Which I would consider a usability bug. Is the only screen in the whole installation in which "abort" means "back" (instead of aborting) and "back" means... well, the same than "abort". :wink: That's why I was talking about desired behavior.